core/publications/default-publication-types.php

```diff
@@ -114,8 +114,8 @@ function tp_register_all_publication_types() {
         array(
             'type_slug'         => 'inproceedings',
             'bibtex_key_ext'    => 'inproceedings',
-            'i18n_singular'     => _x('Inproceedings','Singular form of inproceedings, if it exists','teachpress'),
-            'i18n_plural'       => __('Inproceedings','teachpress'),  
+            'i18n_singular'     => __('Proceedings','teachpress'),
+            'i18n_plural'       => __('Proceedings','teachpress'),
```

Comment:
Hi,
I'm not really understand the change for the labels for inproceedings and incollection. Could you explain the reason for that? With reference to the documentation of biblatex an inproceedings "is an article in a conference proceedings". Also an incollection is a "contribution to a collection which forms a self-contained unit with a distinct author
and title." In short: for me is inproceedings != proceedings and incollection != book section

Reply:
Hi Michael!

You are right, this is a bit messy. On [Wikipedia](https://en.wikipedia.org/wiki/BibTeX#Entry_types), there is a good summary of the BibTeX types. I chose "friendly names" according to this documentation. For `inproceedings`, maybe "Proceedings paper" would be better. For `incollection`, "Book Section" seems reasonable, as the documentation says it is "A part of a book having its own title.". It is up to you ultimately. I wanted to complete the translations.

Fabrice

Reply:
Additionally, please note that I recently pushed more commits to the master branch, and this includes a new feature necessary for a site I am working on. This feature automatically and periodically fetches and downloads `.bib` files from the web and/or Zotero in order to add to the current publications. This feature is necessary because my clients want to work with Zotero. It has been working well.

Reply:
This commit further improves the translations as per our discussion.

https://github.com/winkm89/teachPress/pull/200/commits/abd2f02ba4cc99c0319ca3561ec76822bf0128f0

F